### PR TITLE
Added Models and ViewModels. Tried to create my own mapper

### PR DESCRIPTION
**Pull request by 5Ka-me**

### CODE REVIEW

File: SkyDrive/SkyDrive/ViewModels/InstructorViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; } = null!;
+        public string MiddleName { get; set; } = null!;
+        public string LastName { get; set; } = null!;
+        public int Experience { get; set; }
+
+        public ICollection<EventViewModel>? Events { get; set; }
+
+    }
```
Comment (Genresal) [inline SkyDrive/SkyDrive/ViewModels/InstructorViewModel.cs:13]: Unnecessary whitespace, please remove it.

File: SkyDrive/SkyDrive/ViewModels/MemberViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    public class MemberViewModel
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; } = null!;
+        public string MiddleName { get; set; } = null!;
+        public string LastName { get; set; } = null!;
+
+        public ICollection<EventViewModel>? Events { get; set; }
+
+    }
```
Comment (Genresal) [inline SkyDrive/SkyDrive/ViewModels/MemberViewModel.cs:12]: There too.

**Review by Genresal — APPROVED** [review]

**Review by CrazyKamenwik — APPROVED** [review]

**Review by Finka95 — APPROVED** [review]
